Dockerfile: copy only CA certificate bundle

rockdrilla opened the pull request:
### 1. Why is this pull request needed and what does it do?

Final container image receives a lot of (dangling) symlinks in /etc/ssl/certs/ while only one (regular) file is required.

### 2. Which issues (if any) are related?

None.

### 3. Which documentation changes (if any) need to be made?

None.

### 4. Does this introduce a backward incompatible change or deprecation?

No.

chrisohaver commented on Dockerfile:
```diff
@@ -1,11 +1,10 @@
 
-RUN apt-get update && apt-get -uy upgrade
-RUN apt-get -y install ca-certificates && update-ca-certificates
+RUN apt-get -qq update && apt-get -yyqq install ca-certificates && apt-get clean
```
What is the reason for this change?  I see it removes `apt-get upgrade` adds `apt-get clean`.

rockdrilla replied:
apt-get clean - clean up layer (speeds up build)
apt-get upgrade - doesn't affect layer product (i.e. /etc/ssl/certs/ca-certificates.crt) in any way

chrisohaver commented on Dockerfile:
```diff
@@ -1,11 +1,12 @@
 
-RUN apt-get update && apt-get -uy upgrade
-RUN apt-get -y install ca-certificates && update-ca-certificates
+ENV DEBCONF_NONINTERACTIVE_SEEN=true DEBIAN_FRONTEND=noninteractive DEBIAN_PRIORITY=critical TERM=linux
```
Why are these required?

rockdrilla replied:
These environment variables reduce amount of (useless) information in build logs, mostly debconf tuning.

rockdrilla committed:
Dockerfile: copy only CA certificate bundle

and never more - (dangling) symlinks pointing to /usr/share/ca-certificates/mozilla/ in Debian:stable-slim

Signed-off-by: Konstantin Demin <rockdrilla@gmail.com>

chrisohaver commented on Dockerfile:
```diff
@@ -1,11 +1,18 @@
+SHELL [ "/bin/sh", "-ec" ]
 
-RUN apt-get update && apt-get -uy upgrade
-RUN apt-get -y install ca-certificates && update-ca-certificates
```
Is running `update-ca-certificates` no longer necessary?

rockdrilla replied:
This is done automatically by postinst script in "ca-certificates" package.
